settings/src/script.js

```diff
@@ -156,22 +156,15 @@ function Main( { userId } ) {
 	);
 
 	if ( 'account-status' === screen ) {
-		screenContent = (
-			<div className={ 'wporg-2fa__account-status' }>
-				<AccountStatus />
-			</div>
-		);
+		screenContent = <AccountStatus />;
 	} else if (
 		twoFactorRequiredScreens.includes( screen ) &&
-		userRecord.record[ '2fa_available_providers' ] &&
-		userRecord.record[ '2fa_revalidation' ] &&
-		userRecord.record[ '2fa_revalidation' ].expires_at <= new Date().getTime() / 1000
+		record[ '2fa_available_providers' ].includes( 'Two_Factor_Totp' ) &&
+		record[ '2fa_revalidation' ]?.expires_at <= new Date().getTime() / 1000
```

Comment:
If we're not going to disable the backup codes screen when totp is disabled, we'd need logic to check which provider is enabled, and show the relevant reauth UI; totp or backup code. Something like:

```javascript
const authExpired = record[ '2fa_revalidation' ]?.expires_at <= new Date().getTime() / 1000;

....

} else if (
		record[ '2fa_available_providers' ].includes( 'Two_Factor_Totp' ) && authExpired
) {
                screenContent = (
			<>
				<AccountStatus />
				<RevalidateModalToTP />
			</>
		);
} else if (
		record[ '2fa_available_providers' ].includes( 'Two_Factor_Backup_Codes' ) && authExpired
) {
                screenContent = (
			<>
				<AccountStatus />
				<RevalidateModalBackupCode />
			</>
		);
```

Reply:
> we'd need logic to check which provider is enabled, and show the relevant reauth UI; totp or backup code. 

This should really be handled by the Two-Factor handler I think rather than us (It should display whichever the user logged in with), but, we may need to extend the re-auth period when a login is using the Backup codes, as using up recovery codes every 10 minutes is not ideal.

Perhaps the time should be extended to 1hr for recovery code logins?

Reply:
I like that idea 👍🏻